rebar.lock

```diff
@@ -56,7 +56,7 @@
  {<<"sidejob">>,{pkg,<<"sidejob">>,<<"2.1.0">>},0},
  {<<"ssl_verify_fun">>,{pkg,<<"ssl_verify_fun">>,<<"1.1.6">>},0},
  {<<"syslog">>,{pkg,<<"syslog">>,<<"1.1.0">>},0},
- {<<"template_compiler">>,{pkg,<<"template_compiler">>,<<"2.1.1">>},0},
+ {<<"template_compiler">>,{pkg,<<"template_compiler">>,<<"2.3.0">>},0},
```

Comment:
Maybe also make sense update https://github.com/zotonic/zotonic/blob/master/apps/zotonic_core/rebar.config#L48

Reply:
This is the lock file, in the .config file it says something like `~> 2.1`